beta-src/src/utils/map/WDCountryHighlightFilters.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from "react";
+import webDiplomacyTheme from "../../webDiplomacyTheme";
+
+interface CountryHighlightFilter {
+  countryName: string;
+  floodColor: string;
```

Comment:
I don't think we should pass this, we should reference the country color based on the web dip theme using the country enum.